service/scaladsl/server/src/main/scala/com/lightbend/lagom/scaladsl/server/LagomApplicationLoader.scala

```diff
@@ -245,6 +245,7 @@ abstract class LagomApplication(context: LagomApplicationContext)
     applicationLifecycle.addStopHook(stopHook)
```

Comment:
This is redundant now that `stopHook` is a no-op (it might be good to deprecate the Play `ActorSystemProvider` methods that return it and add new ones that don't).

Reply:
Ref: https://github.com/playframework/playframework/blob/2.7.0-M2/framework/src/play/src/main/scala/play/api/libs/concurrent/Akka.scala#L189-L191

Reply:
Raised https://github.com/playframework/playframework/issues/8534